### ISSUE TITLE
VerifyAndClose

### ISSUE THREAD

**Issue by spydmobile** [PSaaS-Developers/Project_issues#82]

**Comment by spydmobile** [comment]
@RobBryce @nealmcloughlin @BadgerOnABike this was overlooked and needs to be completed as a priority BEFORE final PRE_WISE work is complete.

**Comment by nealmcloughlin** [comment]
Yes, everything here is testing related which can be completed with the next Pre-WISE development build.

**Comment by spydmobile** [comment]
@nealmcloughlin and @BadgerOnABike this is old business that we need to complete. can we complete this work with the builds we have?
